src/hapi-i18next.ts

```diff
@@ -43,7 +43,7 @@ export var register: HapiPluginRegister = function (server, options: any, next):
 	 * @description
 	 */
 	server.method('i18n.getInstance', (): any => {
-		return i18n;
+		return require('i18next');
```

Comment:
We still need to load i18next on the top of the file for internal usage, but node modules get cached and we need to give hapi a fresh instance to avoid sharing the cached language between different users.